src/data/agile-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const agileQuiz = [
+  {
+    Question: "Agile Software Development is based on",
+    Answer: "Both Incremental and Iterative Development",
+    Distractor1: " Incremental Development",
+    Distractor2: " Iterative Development",
+    Distractor3: "Linear Development",
+    Explanation:
+    "The software is developed in increments with the customer specifying the requirements to be included in each increment and the highest priority is to satisfy the customer through early and continuous delivery of valuable software. They are iterative because they work on one iteration followed by improvements in next iteration",
+    Link: "https://www.sanfoundry.com/software-engg-mcqs-agile-software-development/"
+  },
```

Comment:
we already have a similar question in the file so we will not be accepting this one.
Please come up with another question to contribute that is not already in the file 👍